uint8 STOP=0
uint8 PAUSE=1
uint8 PLAY=2
uint8 command
---
bool success